apps/portal/src/components/color-palette.tsx

```diff
@@ -0,0 +1,77 @@
+import * as styles from "@harnessio/core-design-system/styles-esm";
+import { Tooltip } from "@harnessio/ui/components";
+
+interface Color {
+  name?: string;
+  $value?: string;
+  [key: string]: unknown;
+}
+
+export function ColorPalette() {
+  const shades = Object.keys(styles.colors.blue);
+  const colors = Object.keys(styles.colors).filter(
+    (color) => color !== "pure",
+  ); /** Excluding "pure" colors intentionally */
+
+  return (
+    <div className="grid rounded-lg border border-borders-4 bg-background-4 px-6 pt-1 pb-6 mt-4 shadow-md h-fit w-[800px]">
+      {/* Shades */}
+      <div className="flex">
```

Comment:
We should be able to get rid of the flex layout and put the shades into the grid

Reply:
Done, removed `flex` and added `grid`.